schema/url/sequence.raml

```diff
@@ -26,10 +26,6 @@ types:
     type: string
     enum: ['bed', 'gff3']
 
-  SequenceType:
```

Comment:
I do think the new sequences should be added with a config change, not code change - it's how we're setting up the service, e.g one genomic and one protein sequence in test, and a few different ones in production.

The options are "is stranded" which tells the service what to do about negative strand entries in .bed. The other options limits for throughput we could potentially have them global rather than different for each request!